client/RenderService.d.ts

```diff
@@ -0,0 +1,14 @@
+/* Typescript type definitions for RenderService */
+
+declare type Position = [x: number, y: number]
+
+declare namespace RenderService {
+	function circle(pos: Position, radius: number, color?: string): void;
```

Comment:
so fun fact -- these "Position" inputs allow both:
```
[x, y]
aSpiritObject
```
Is it possible to enable that too? Specifically, it just looks for the "position" property on the input object. (very janky i know)

Reply:
Didn't know it did that, thanks! I'll update it